client/src/pages/Restaurant/FoodList/index.js

```diff
@@ -1,5 +1,27 @@
-function FoodList() {
-  return <div>FoodList</div>;
+import FoodCard from "./FoodCard";
+
+const classes = {
+  foodList: {
+    display: "flex",
+    flexWrap: "wrap",
+    justifyContent: "center",
+    gap: "1rem",
+  },
+};
+
+function FoodList({ foods }) {
+  return (
+    <div style={classes.foodList}>
+      {foods.map((food) => (
+        <FoodCard
```

Comment:
Missing key here. If you inspect, you should expect React complaining about it. Just use a temp key for now. In the future, we'll use the food Id from the database (need to fix the food presenter from the server)